zproject/backends.py

```diff
@@ -1470,6 +1483,133 @@ class GitLabAuthBackend(SocialAuthMixin, GitLabOAuth2):
     # we just use the primary email address, which is always verified.
     # (No code is required to do so, as that's the default behavior).
 
+@external_auth_method
+class AppleAuthBackend(SocialAuthMixin, AppleIdAuth):
+    sort_order = 10
+    name = "apple"
+    auth_backend_name = "Apple"
+    full_name_validated = True
```

Comment:
This probably deserves a comment that full_name isn't available on repeat logins, but when it is, it is validated.

Reply:
Added.